src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration.component.html

```diff
@@ -3,7 +3,17 @@
           [columns]="columns"
           selectionType="single"
           (updateSelection)="updateSelection($event)">
-  <div class="table-actions form-inline">
+  <div class="table-actions">
```

Comment:
You should use the `table actions component` and use role management functionalities to only show the edit button if needed.